payments-ui-core/src/main/java/com/stripe/android/paymentsheet/forms/PaymentMethodRequirements.kt

```diff
@@ -289,3 +289,9 @@ internal val BoletoRequirement = PaymentMethodRequirements(
     siRequirements = setOf(Delayed),
     confirmPMFromCustomer = true,
 )
+
+internal val KonbiniRequirement = PaymentMethodRequirements(
+    piRequirements = setOf(Delayed),
+    siRequirements = setOf(Delayed),
```

Comment:
Is this correct? I thought SI is not supported?

Reply:
Good catch. Thanks!